CLI.md

```diff
@@ -1053,11 +1053,11 @@ List the blobs of the database
 
 
 
-## `linera storage list_root_keys`
+## `linera storage list_chain_ids`
 
-List the root keys of the database
+List the chain ids of the database
```

Comment:
IIRC @afck prefers `IDs` ?

Reply:
Ok, changed (and also in some other places.